preload_qsos_dr7.m

```diff
@@ -87,10 +88,14 @@
   end
 
   all_normalizers(i) = this_median;
-
+ 
   this_flux           = this_flux           / this_median;
   this_noise_variance = this_noise_variance / this_median^2;
  
+
+  % ind = (this_rest_wavelengths >= loading_min_lambda) & ...
+  %       (this_rest_wavelengths <= loading_max_lambda);
```

Comment:
better to uncomment these two lines since `ind` here is the indices you want to load. If they are commented, you will read the ind from line 80, which has excluded pixel_mask. But we want to load a spectrum “with” pixel masks. Also make loading_min_lambda and loading_max_lambda wider than min_lambda and max_lambda.

Reply:
That's true!